src/components/InjectedComponents/ImmersiveGuide/SetupGuide.tsx

```diff
@@ -506,6 +507,7 @@ export function SetupGuide(props: SetupGuideProps) {
                 _wallet_is_default: true,
```

Comment:
`Services.Identity.setupPersona()` set the `uninitialized` to `false`, so we should `MessageCenter.emit('identityUpdated', undefined)` after that finish